config/routes.rb

```diff
@@ -131,7 +131,11 @@
 
   get 'login' => 'users#login', as: 'login'
   get 'logout' => 'users#logout'
-  get 'urs_callback' => 'oauth_tokens#urs_callback'
+  get 'prompt_urs_association' => 'users#prompt_urs_association'
+  get 'confirm_urs_association' => 'users#confirm_urs_association'
+  post 'associate_urs_and_launchpad_ids' => 'users#associate_urs_and_launchpad_ids'
+  get 'urs_login_callback' => 'oauth_tokens#urs_login_callback'
```

Comment:
I assume you updated all of the URS applications for the new callback uris? SIT UAT and PROD?

Reply:
not yet in UAT and PROD (I will check if I have access). I did update it in SIT, that was one of the hiccups this morning.